PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetEnergyFlow.cxx

```diff
@@ -398,7 +400,7 @@ void AliAnalysisTaskEmcalJetEnergyFlow::AllocateEnergyflowHistograms(){
 
 
   Int_t Pair_number = 3;
- if (isMCprod)Int_t Pair_number = fJetCollArray.GetEntries()/2-1;
+ if (IsMCprod)Int_t Pair_number = (fJetCollArray.GetEntries()/2)-1;
  else Int_t Pair_number = fJetCollArray.GetEntries()-1; 
```

Comment:
This won't do what you want. Your redefinition shadows the initial definition. As a result, `Pair_number` will always be 3.

Reply:
Okay, I will fix that. Thanks for the comment.

Reply:
You didn't fix the issue. See this example:

```c++
int i = 3;
if (true) int i = 10;
// This will always print 3 because you redefined `i`
std::cout << "i: " << i << "\n";
```

You can fix it by removing the `int` from the second line. So it would be: `if (true) i = 10;`

Reply:
I see, that was a very silly mistake. Now it should be okay, sorry for the email/notification spam